live_validation_scripts/asana_reports.mjs

```diff
@@ -69,207 +98,304 @@ function readAsanaNotifyFile() {
 
+    let hasErrors = false;
+
+    // Load the assana access token
+    try {
+        token.accessToken = fs.readFileSync('/etc/ddg/env/ASANA_DAX_TOKEN', 'utf8');
+    } catch (error) {
+        console.error('Error reading access token from file:', error);
+        process.exit(1);
+    }
+
     // Load user map
-    if (!fs.existsSync(argv.userMapFile)) {
-        console.error(`User map file ${argv.userMapFile} does not exist!`);
+    try {
+        userMap = yaml.load(fs.readFileSync(argv.userMapFile, 'utf8'));
+    } catch (error) {
+        console.error(`Error reading ${argv.userMapFile}:`, error);
         process.exit(1);
     }
-    userMap = yaml.load(fs.readFileSync(argv.userMapFile, 'utf8'));
 
-    // Save the directory path and load the asana notify file
+    // Load the asana notify file
     const success = readAsanaNotifyFile(dirPath);
-
     if (!success) {
         console.log(`Error: Failed to read asana notify file ${dirPath}/asana_notify.json`);
         process.exit(1);
     }
 
-    // Load the pixelsWithErrors object
+    // Load the pixelsWithErrors object produced by validate_live_pixel.mjs
     const pixelsErrorsPath = fileUtils.getPixelErrorsPath(dirPath);
     console.log(`Pixel with errors path from fileUtils: ${pixelsErrorsPath}`);
 
-    if (fs.existsSync(pixelsErrorsPath)) {
-        try {
-            const pixelErrorsData = fs.readFileSync(pixelsErrorsPath, 'utf8');
-            pixelsWithErrors = JSON.parse(pixelErrorsData);
+    try {
+        const pixelErrorsData = fs.readFileSync(pixelsErrorsPath, 'utf8');
+        pixelsWithErrors = JSON.parse(pixelErrorsData);
 
-            const pixelNames = Object.keys(pixelsWithErrors);
-            console.log(`Successfully loaded pixel with errors object with ${pixelNames.length} pixels`);
-        } catch (error) {
-            console.error(`Error parsing pixel with errors JSON:`, error);
-        }
-    } else {
-        console.log(`Pixel errors file not found at: ${pixelsErrorsPath}`);
+        const pixelNames = Object.keys(pixelsWithErrors);
+        console.log(`Successfully loaded pixel with errors object with ${pixelNames.length} pixels`);
+    } catch (error) {
+        console.error(`Error parsing pixel with errors JSON:`, error);
+        process.exit(1);
     }
 
     const numPixelsWithErrors = Object.keys(pixelsWithErrors).length;
     console.log('Final number of pixel with errors keys (object):', numPixelsWithErrors);
 
-    // Build ownersWithErrors from pixelsWithErrors
-    const ownersSet = new Set();
-    for (const pixel of Object.values(pixelsWithErrors)) {
+    // Build ownerToPixelsMap from pixelsWithErrors
+    // We could modify validate_live_pixel.mjs to export this format
+    ownerToPixelsMap = {};
+    for (const [pixelName, pixel] of Object.entries(pixelsWithErrors)) {
         if (pixel.owners) {
-            pixel.owners.forEach((owner) => ownersSet.add(owner));
+            pixel.owners.forEach((owner) => {
```

Comment:
Won't this assign the same pixel more than once if there are multiple owners? I'd just assign to the first owner. Otherwise we risk 2+ people spending their time fixing the same thing